store/sqlstore/group_store.go

```diff
@@ -1016,6 +1016,8 @@ func (s *SqlGroupStore) GetGroups(page, perPage int, opts model.GroupSearchOpts)
 		`, opts.NotAssociatedToChannel)
 	}
 
+	groupsQuery = groupsQuery.Where("g.DeleteAt = 0")
```

Comment:
I was curious if we could do this instead of reassigning it later in the code, could we do it in the beginning where we create the `groupsQuery`? Changing `line 966` to
```suggestion
		groupsQuery := s.
		getQueryBuilder().
		Select("g.*").
		From("UserGroups g").
		Where("g.DeleteAt = 0") // <--- Appended this line to the Query
		Limit(uint64(perPage)).
		Offset(uint64(page * perPage)).
		OrderBy("g.DisplayName").
```
Thoughts? It doesn't matter either way but curious what you think?

Reply:
I initially did that but that actually required me adding the same line in two places line 966 and also inside the `if opts.IncludeMemberCount {` 

I have now refactored it slightly to remove some duplication of the query logic allowing me to add the `Where("g.DeleteAt = 0"). ` closer to the start of the method while also only having to add it in one place